docs/release-notes/8.16.asciidoc

```diff
@@ -29,6 +29,19 @@ On November 12, 2024, it was discovered that manually running a custom query rul
 ====
 // end::known-issue[]
 
+// tag::known-issue[]
+[discrete]
+.Installing an {elastic-defend} integration or a new agent policy upgrades all the installed prebuilt rules
```

Comment:
Should we mention overwriting actions and exceptions in the header?

Reply:
@approksiu how about this?  
> Installing an {elastic-defend} integration or a new agent policy overwrites user-added rule actions and exceptions